test/tools/ossfuzz/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ if (OSSFUZZ)
             protobuf.a
 )
     set_target_properties(yul_proto_diff_custom_mutate_ossfuzz PROPERTIES LINK_FLAGS ${LIB_FUZZING_ENGINE})
+    target_compile_options(yul_proto_diff_custom_mutate_ossfuzz PUBLIC ${COMPILE_OPTIONS} -Wno-sign-conversion)
```

Comment:
Is there a way to add compile options per file. What this and the previous lines do is disable signed conversion checks for *all* source files comprising the target. Ideally, we'd only want to disable signed conversion check for auto generated protobuf header files and not for fuzzer source files (e.g., protoToYul.cpp) that are maintained in this repo.

Reply:
Approving anyway cos this is a more complex issue for which I created a different ticket and assigned myself :)

(see https://github.com/ethereum/solidity/issues/9395)